app/controllers/sessions_controller.rb

```diff
@@ -2,7 +2,9 @@ class SessionsController < ApplicationController
   skip_before_action :authenticate_user!, only: %i[new callback failure]
   before_action :redirect_to_after_sign_in_path, only: %i[new], if: :user_signed_in?
 
-  def new; end
+  def new
+    render "placements/pages/start" if current_service == :placements
```

Comment:
I've [asked about the difference](https://ukgovernmentdfe.slack.com/archives/C04MLBVP876/p1714750236261039) between Funding Mentors and School Placements, with regards to the start page vs. login page, and we're going to stick with the same approach as FM for now.

In other words: we'll still have that interstitial `/sign-in` page with the "Sign in using DfE Sign-in" button.

I think this should simplify the implementation of this start page, because we should be able to get rid of conditional logic like this.

I'd like to circle back round to this at a later date and see if we can remove that intermediary "sign in" page in a different way. Because once our service start page is hosted on GOV.UK, we won't be able to use a form POST for the "start" button. For now it's [on the backlog](https://trello.com/c/zEljg2Hv/141-remove-the-intermediary-sign-in-page-from-the-service).